src/components/SharedComponents/NavBar/NavButton.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import { Image, Pressable } from "components/styledComponents";
+import { t } from "i18next";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { IconButton } from "react-native-paper";
+import colors from "styles/tailwindColors";
+
+type Props = {
+  id: string,
+  icon: any,
+  onPress: any,
+  img?: string,
+  accessibilityLabel: string,
+  accessibilityRole?: string,
+  active: boolean,
+  size: number,
+};
+
+const imgStyle = ( size, active ) => StyleSheet.create( {
+  img: {
+    borderWidth: active ? 3 : 0,
+    height: size,
+    width: size,
+    borderColor: colors.inatGreen
+  }
```

Comment:
I'd rather have used tailwind here...

Unfortunately setting the border width using class `border-[3px]` didn't work on android

setting height via classes didnt work on either platform

and you can't interpolate class's in tailwind